src/main/java/com/daitangroup/vehicleImplementations/VWSaveiroCross.java

```diff
@@ -0,0 +1,49 @@
+package com.daitangroup.vehicleImplementations;
+
+import com.daitangroup.vehiclesTypes.UtilityVehicle;
+
+import java.util.Objects;
+
+public class VWSaveiroCross extends UtilityVehicle {
+
+   private String engineType = "1.6 16V engine";
+   private String torque = "10,51 kgfm/litro";
+   private boolean tracctionControl = true;
+   private boolean courtesyLight = true;
+   private String interiorDescription = "All black with a big multimedia screen.";
```

Comment:
Does this variable changes its content? 
If not, should be a final.

Reply:
His cars are tunnable!